server/src/routes/Auth.route.js

```diff
@@ -5,10 +5,10 @@ import asyncErrorHandler from '../middlewares/asyncErrorHandler';
 
 const router = Router();
 
-const { signup } = AuthController;
-const { validateSignup } = AuthValidation;
+const { signup, login } = AuthController;
+const { validateSignup, validateLogin } = AuthValidation;
 
 
 router.post('/signup', validateSignup, asyncErrorHandler(signup));
-
+router.post('/login', validateLogin, asyncErrorHandler(login));
```

Comment:
Do you now properly understand `asyncErrorHandler`?

Reply:
not really...

Reply:
you need to